src/utils/displayAlertBar.tsx

```diff
@@ -2,6 +2,6 @@ import React, { ReactElement } from "react";
 
 import { AlertBar } from "@src/components/display/fewlines/AlertBar/AlertBar";
 
-export function displayAlertBar(text: string): ReactElement {
+export function displayAlertBar(text: string | undefined): ReactElement {
```

Comment:
In what cases could this text be undefined?

Reply:
It used to be possible a few commits ago when I was calling the function pretty much this way: `displayAlertBar(Cookie.get("flashMessage");`, the parameter I was using would have return either a `string` or `undefined`.
Therefore, I had to adapt the type of the parameter `text` of the function `displayAlertBar()`, and likewise for the property `text` of the component `<AlertBar />`.
Now that I'm using `getFlashMessage()` function that returns `string | void`, I no longer need these adaptations, so I brought them to their initial types 7abd3cd